src/pages/Orders/index.jsx

```diff
@@ -0,0 +1,143 @@
+import filterIcon from "../../assets/icons/filter.svg"
+import { RxCaretDown } from "react-icons/rx";
+import { RiDownload2Fill } from "react-icons/ri";
+import { PiCaretRight } from "react-icons/pi";
+import { PiCaretLeft } from "react-icons/pi";
+import Searchbar from "../../components/Searchbar";
+
+
+
+
+const Orders = () => {
+
+
+    const ORDER_DATA = [
```

Comment:
It is only a mock data so store this data in separate file `/src/assets/mock-data/...`